inspire/modules/formatter/views.py

```diff
@@ -47,3 +47,28 @@ def get_latex(recid, latex_format):
     record = get_record(recid)
     latex = Latex(record, latex_format).format()
     return jsonify({"result": latex, "recid": recid})
+
+
+@blueprint.route("/download-bibtex/<int:recid>")
+def get_bibtex_file(recid):
+    record = get_record(recid)
+    results = Bibtex(record).format()
+    generator = (cell for row in results
```

Comment:
It returns the text ready, so if you download it and open it looks like
this:
[image: Ενσωματωμένη εικόνα 1]

2015-09-02 13:08 GMT+02:00 Javier Martin Montull <notifications@github.com>:

> In inspire/modules/formatter/views.py
> <https://github.com/inspirehep/inspire-next/pull/333#discussion_r38520497>
> :
>
> > @@ -47,3 +47,28 @@ def get_latex(recid, latex_format):
> >      record = get_record(recid)
> >      latex = Latex(record, latex_format).format()
> >      return jsonify({"result": latex, "recid": recid})
> > +
> > +
> > +@blueprint.route("/download-bibtex/<int:recid>")
> > +def get_bibtex_file(recid):
> > +    record = get_record(recid)
> > +    results = Bibtex(record).format()
> > +    generator = (cell for row in results
>
> What is the format returned by Bibtex(record).format() ? It does not
> return the text ready to be returned?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/inspirehep/inspire-next/pull/333/files#r38520497>.
>